scripts/common/rook.sh

```diff
@@ -100,9 +100,16 @@ function remove_rook_ceph() {
     printf "Data within /var/lib/rook, /opt/replicated/rook and any bound disks has not been freed.\n"
 }
 
-# scale down prometheus, move all 'rook-ceph' PVCs to 'longhorn', scale up prometheus
-function rook_ceph_to_longhorn() {
-    report_addon_start "rook-ceph-to-longhorn" "v1"
+# scale down prometheus, move all 'rook-ceph' PVCs to provided storage class, scale up prometheus
+# Supported storage class migrations from ceph are: 'longhorn' and 'openebs'
+function rook_ceph_to_sc_migration() {
+    local destStorageClass=$1
+    report_addon_start "rook-ceph-to-${destStorageClass}-migration" "v2"
```

Comment:
this should not report something with a variable in it as destStorageClass is configurable by the vendor. you should get the provisioner and use that

Reply:
good point. I'll change this.